scripts/install-worker.sh

```diff
@@ -122,15 +122,25 @@ sudo mv $TEMPLATE_DIR/iptables-restore.service /etc/eks/iptables-restore.service
 if [[ "$BINARY_BUCKET_REGION" != "us-iso-east-1" && "$BINARY_BUCKET_REGION" != "us-isob-east-1" ]]; then
   # https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html
   echo "Installing awscli v2 bundle"
-  AWSCLI_DIR=$(mktemp -d)
-  curl \
-    --silent \
+  AWSCLI_DIR="${AWSCLI_DIR:-""}"
+  if [[ -n $AWSCLI_DIR ]]; then
+    AWSCLI_DIR=$AWSCLI_DIR
```

Comment:
This is a no-op.

Reply:
The line does work, the default behaviour goes to mktemp because AWSCLI_DIR is "" by default. If you parse the environment variable then it uses the directory you parse

Reply:
I'm referring to this:
```
AWSCLI_DIR=$AWSCLI_DIR
```

Assigning a variable to itself has no effect.

Reply:
Ah i see, yes. I'll remove that